docs/rule/builtin-component-arguments.md

```diff
@@ -13,8 +13,17 @@ This rule warns about `Input` component invocations that use the following attri
 * `type`
 * `value`
 
+The builtin `Textarea` component has several arguments that match attributes
+of the lower-case `textarea` HTML element. These arguments should be set via e.g.
```

Comment:
I know this is pedantic, but `value` is **not** a valid attribute of `<textarea></textarea>`. It is however a valid **property** of the resulting DOM element. 😭 

```suggestion
The builtin `Textarea` component has several arguments that match properties
of the lower-case `textarea` HTML element. These arguments should be set via e.g.
```